sympy/sets/sets.py

```diff
@@ -1102,6 +1127,11 @@ def _sup(self):
         from sympy.functions.elementary.miscellaneous import Max
         return Max(*[set.sup for set in self.args])
 
+    @property
+    def is_empty(self):
+        if any(set.is_empty == False for set in self.args):
+            return False
```

Comment:
This is Union.is_empty so maybe it could be fuzzy_and

Reply:
I'll change it. Thanks.